src/status_im/chat/screen.cljs

```diff
@@ -44,20 +29,28 @@
         [react/text {:style style/add-contact-text}
          (i18n/label :t/add-to-contacts)]]])))
 
+(defn- on-options [chat-id chat-name group-chat?]
+  (list-selection/show {:title   chat-name
+                        :options (actions/actions group-chat? chat-id)}))
+
 (defview chat-toolbar []
-  (letsubs [show-actions? [:get-current-chat-ui-prop :show-actions?]
-            accounts      [:get-accounts]
-            creating?     [:get :accounts/creating-account?]]
+  (letsubs [accounts    [:get-accounts]
+            creating?   [:get :accounts/creating-account?]
+            group-chat? [:chat :group-chat]
+            chat-name   [:chat :name]
+            chat-id     [:chat :chat-id]]
```

Comment:
Why so many subscriptions, isn't better and more effective to just use `[:get-current-chat]` and destructure the necessary properties ?
We are eventually trying to get rid of the general `:get/:set` subs and events as they are anti-pattern (clearly stated in `re-frame` guide), so it would be best to not add them in new code.

Reply:
Didn't change the original design as this is more a UI PR.
Do you think we have all details with `get-current-chat` ?

Reply:
No problem with that, that's why I didn't requested changes, just mentioned it.
It would also save us 2 lines of code, being:
```clj
{:keys [group-chat name chat-id} [:get-current-chat]
```
Instead of those three lines.